### ISSUE TITLE
Move HomepageTimer Clock Config into Props

### ISSUE THREAD

**Issue by awanlin** [backstage/backstage#7164]
## Feature Suggestion

While working on getting the `HomepageTimer` into the new composable Home Page it was [suggested](https://github.com/backstage/backstage/pull/7029#pullrequestreview-744807795) by @iamEAP that we move the clock config into props and out of the `app-config.yaml`

## Possible Implementation

This part I'm not totally sure about but would like to hammer something out in this issue. I'm happy to contribute this once we have a solution

## Context

This would simplify the setup of the `HomepageTimer` for integrators and reduce the amount of config in the `app-config.yaml`

**Comment by awanlin** [comment]
@iamEAP do you have some ideas as to how we would do this? I'm happy to run with that!

**Comment by iamEAP** [comment]
Hey @awanlin!  Thanks for opening this follow-up.

I imagine the meat of this would just be refactoring the `<HomepageTimer />` in `@backstage/core-components` to optionally take in a `clocks` prop that looks a lot like the `homepage.clocks` config that currently exists (e.g. an array of objects with keys `label` and `timezone`).  ...For backward compatibility, you could continue supporting the config-driven approach, but ignore it if props are provided?

Once that's done, I suppose you could...  Remove the `homepage.clocks` config in the monorepo's `app-config.yaml`, and transfer the values to props in the app package's composed homepage.

**Comment by iamEAP** [comment]
Better yet!  This may be an opportunity to remove this from the `core-components` package, and let it sit in the `home` plugin.  So instead of refactoring, you could probably just slap a `@deprecated` on the original component, and then re-implement in the home plugin, only accepting props (no config support).

**Comment by awanlin** [comment]
Yes, that makes a lot of sense, thanks for the tips. I'll take a crack at this as I have time. I also would like to add the fun multi-language Welcome message back as something optional as we loved that feature!
